releases/release-7.5.7.md

```diff
@@ -0,0 +1,228 @@
+    - (dup): release-8.5.3.md > 错误修复> TiKV - 修复慢日志中 `StoreMsg` 日志信息存在误导性描述的问题 [#18561](https://github.com/tikv/tikv/issues/18561) @[LykxSassinator](https://github.com/LykxSassinator)
+    - (dup): release-9.0.0.md > 错误修复> TiKV - 修复错误的线程内存监控指标 [#18125](https://github.com/tikv/tikv/issues/18125) @[Connor1996](https://github.com/Connor1996)
+    - (dup): release-8.5.3.md > 错误修复> TiKV - 修复 TiKV 在正常退出时未能终止正在进行的手动 compaction 任务的问题 [#18396](https://github.com/tikv/tikv/issues/18396) @[LykxSassinator](https://github.com/LykxSassinator)
+
++ PD <!--tw@qiancai: 5 notes-->
+
+    - 修复反复修改 `split-merge-interval` 的值，可能导致该配置不生效的问题 [#8404](https://github.com/tikv/pd/issues/8404) @[lhy1024](https://github.com/lhy1024)
+    - 修复 `lease` 默认值未被正确设置的问题 [#9156](https://github.com/tikv/pd/issues/9156) @[rleungx](https://github.com/rleungx)
+    - 修复了 TiDB Dashboard 导致的 pd goroutine 泄露问题 [#9402](https://github.com/tikv/pd/issues/9402) @[baurine](https://github.com/baurine)
+    - 修复新上线节点可能无法被调度问题 [#9145](https://github.com/tikv/pd/issues/9145)@[bufferflies](https://github.com/bufferflies)
+    - 修复开启 tidb_enable_tso_follower_proxy 后，tso 服务不可用问题 [#9188]https://github.com/tikv/pd/issues/9188 @[Tema](https://github.com/Tema)
+
++ TiFlash <!--tw@qiancai: 5 notes-->
+
+    - note [#issue](https://github.com/pingcap/tiflash/issues/${issue-id}) @[贡献者 GitHub ID](https://github.com/${github-id})
+    - 修复在运行 `IMPORT INTO` 或 `BR restore` 的时候，部分 SST 文件可能被意外删除导致 TiFlash panic 的问题 [#10141](https://github.com/pingcap/tiflash/issues/10141) @[CalvinNeo](https://github.com/CalvinNeo)
+    - (dup): release-8.5.3.md > 错误修复> TiFlash - 修复创建 `((NULL))` 形式的表达式索引会导致 TiFlash panic 的问题 [#9891](https://github.com/pingcap/tiflash/issues/9891) @[JaySon-Huang](https://github.com/JaySon-Huang)
+    - (dup): release-8.5.2.md > 错误修复> TiFlash - 修复 TiFlash 处理带有不规则 key-range 的 Region snapshot 时可能 panic 的问题 [#10147](https://github.com/pingcap/tiflash/issues/10147) @[JaySon-Huang](https://github.com/JaySon-Huang)
+    - (dup): release-8.5.2.md > 错误修复> TiFlash - 修复当集群的表中存在大量 `ENUM` 类型列时，TiFlash 的内存可能会大量消耗问题 [#9947](https://github.com/pingcap/tiflash/issues/9947) @[JaySon-Huang](https://github.com/JaySon-Huang)
+    - (dup): release-8.5.2.md > 错误修复> TiFlash - 修复插入单行超过 16 MiB 的数据后，TiFlash 可能无法正常重启的问题 [#10052](https://github.com/pingcap/tiflash/issues/10052) @[JaySon-Huang](https://github.com/JaySon-Huang)
+    - (dup): release-8.5.3.md > 错误修复> TiFlash - 修复 Resource Control Low Token Signal 丢失导致查询被限速的问题 [#10137](https://github.com/pingcap/tiflash/issues/10137) @[guo-shaoge](https://github.com/guo-shaoge)
+    - (dup): release-8.5.2.md > 错误修复> TiFlash - 修复执行包含 `GROUP BY ... WITH ROLLUP` 的 SQL 语句时，可能会出现 `Exception: Block schema mismatch` 报错的问题 [#10110](https://github.com/pingcap/tiflash/issues/10110) @[gengliqi](https://github.com/gengliqi)
+    - 降低因为 SSL 重新加载导致 TiFlash crash 的可能性(#8535) [#8535](https://github.com/pingcap/tiflash/issues/8535) @[windtalker](https://github.com/windtalker)
```

Comment:
This issue has been already covered in [v7.5.2 release notes](https://docs.pingcap.com/zh/tidb/dev/release-7.5.2/). The corresponding PR is merged in v7.5.2.
```suggestion
```